cmd/node/main.go

```diff
@@ -1446,6 +1450,7 @@ func createApiResolver(
 	gasSchedule map[string]map[string]uint64,
 	economics *economics.EconomicsData,
 	messageSigVerifier vm.MessageSignVerifier,
+	txTypeHandler process.TxTypeHandler,
```

Comment:
txtypehandler can be created here, inside the method.

Reply:
done.